docs/userguide/transforms/general/rules.md

```diff
@@ -24,7 +24,7 @@ in building transforms, see "Guidelines for Designing Script Transliterations"
 (§) in the [General Transforms](index.md) chapter. For external sources for
 script transforms, see Script Transliterator Sources (§) in that same chapter)
 
-* See *[*Properties and ICU Rule Syntax*](../../strings/properties.md) *for
+:point_right: **Note**: See *[*Properties and ICU Rule Syntax*](../../strings/properties.md) *for
```

Comment:
"notes" on this page are also missing the `>`

Reply:
Fixed